crates/xilem_html/src/element/mod.rs

```diff
@@ -32,7 +33,7 @@ impl<El, ViewSeq> Element<El, ViewSeq> {
         impl<'a, El, VS> fmt::Debug for DebugFmt<'a, El, VS> {
             fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
                 write!(f, "<{}", self.0.name)?;
-                for (name, value) in &self.0.attributes {
+                for (name, value) in self.0.attributes.iter() {
```

Comment:
Would it be nicer to avoid this particular change and have an `IntoIterator` implement for `VecMap`?

Reply:
I have thought about it, but since `impl` for associated types are not stabilized yet it's not completely trivial as there's a closure in `VecMap::iter` (if at all possible? Does someone know how that could be done with closures?). I avoided more complex code in the `VecMap::iter()` just to make `IntoIterator` possible (as `vecmap.iter()` is a minor drawback IMO compared to `&vecmap`).